### DISCUSSION TITLE
A bug of tool hex-decode "UTF8 charset decode wrong"

### DISCUSSION THREAD

**Discussion by Gearace** [he3-app/he3-feedback#199] [Bug Report]
### Version
hex-decode 1.0.2

### Environment
Windows v1.3.7

### Steps to reproduce
input "e4bda0e5a5bd"

### What is expected?
UTF8 charset "你好"

### What is actually happening?
It's decode to "ä½ å¥½"

**Comment by lyzhang1999** [comment]
“你好” Hex 编码后的字符串是“4F60597D”，请问有其他在线工具方便我们验证吗？
